src/deprecations.jl

```diff
@@ -123,6 +123,10 @@ Base.@deprecate_binding StdSpec StdAffineScheme
 # deprecated for 1.1
 @deprecate morphism_of_projective_schemes morphism
 
+# deprecated from src/Rings/mpoly-local.jl
+@deprecate minimal_generators minimal_generating_set
+Base.@deprecate_binding MPolyRingElemLoc MPolyLocRingElem
```

Comment:
Please put them to the bottom of the file to keep it easier to track historical changes